src/components/app.tsx

```diff
@@ -17,19 +19,31 @@ import { cx } from 'src/lib/cx';
 import { getIpcApi } from 'src/lib/get-ipc-api';
 import { UserSettings } from 'src/modules/user-settings';
 import { WhatsNewModal, useWhatsNew } from 'src/modules/whats-new';
+import 'src/index.css';
 
 export default function App() {
 	useLocalizationSupport();
 	const { needsOnboarding } = useOnboarding();
 	const { isSidebarVisible, toggleSidebar } = useSidebarVisibility();
 	const { showWhatsNew, closeWhatsNew } = useWhatsNew();
+	const { i18n } = useI18nData();
 
 	useEffect( () => {
 		void getIpcApi().setupAppMenu( { needsOnboarding } );
 	}, [ needsOnboarding ] );
 
 	return (
 		<>
+			<DynamicStylesheet
+				id="wordpress-components-style"
+				href={
+					i18n?.isRTL()
+						? '/main_window/styles/wordpress-components-style-rtl.css'
+						: '/main_window/styles/wordpress-components-style.css'
+				}
+			/>
+			<DynamicStylesheet id="main-window-style" href={ '/main_window.css' } />
```

Comment:
I'm not using `Helmet` because it doesn't preserve the order of stylesheets after the href has changed.
See https://github.com/staylor/react-helmet-async/issues/187#issuecomment-2903361755